packages/cascara/src/ui/Table/fixtures/TableDX.test.js

```diff
@@ -9,6 +9,7 @@ describe('Table DX', () => {
   test(`should show a loading state if the 'data' prop is 'undefined' or 'null'`, () => {
     const { baseElement } = render(loading);
     // This technically captures the entire fixture in a snapshot. We should consider if we want to only select the element within for a snapshot or not.
+    // update: by default, snapshots are wrapped by a `<body>` tag, which doesn't cause snapshots to grow signigicantly
```

Comment:
@lordpixel note that the reason I am pointing this out in the DX test is that if someone needs to change the copy in the fixture, the snapshot would technically fail and need to be redone. This is why I am thinking we might want to explicitly use the RTL `within` selector to find our specific component in the fixture and base our snapshots off that only.

For now this is not happening so if the description inside the Fixture changes, the snapshot will need to get updated. Maybe that is okay for now, but it is something to be aware of.

Reply:
@brianespinosa I thought about this when we started to use fixtures for unit tests... It seems to me like we are bloating the tests and then wanting for them to run blazing fast... Aren't we shooting ourselves in the foot with this?

Reply:
@lordpixel what are you saying is bloating the tests? The use of `within` from RTL, or the fact that `render` might contain an h2 or p tag with description text?

Reply:
@brianespinosa what I am saying is that things like `faker` or `cosmos fixtures` might be overkill for tests and are just adding even more load, hence tests run sluggier.